adapters/openweb/openwebtest/supplemental/explicit-dimensions.json

```diff
@@ -0,0 +1,59 @@
+{
+  "mockBidRequest": {
+    "id": "test-request-id",
+    "imp": [
+      {
```

Comment:
I am not sure I understand what the name of this test `explicit-dimensions` mean.. Can you please elaborate on what exact functionality are you trying to test with this test case?